.vsts-dotnet-ci.yml

```diff
@@ -35,6 +35,11 @@ jobs:
       mergeTestResults: true
     continueOnError: true
     condition: always()
+  - task: CmdLine@2
```

Comment:
As I understand it, you have it set to copy the results that came back from the verify tests when they fail (nice!) but it only applies to windows full, windows core, and macos. Any reason you chose those three and not others? If it's so hard to get these back, it also makes me wonder about the relative value of verify vs. shouldly, but it isn't my place to opine on that.

Reply:
Because we have some `UniqueForOSPlatform()` tests, there are three sets of snapshots checked in, for Windows, macOS, and Linux. It's nontrivial to produce the baselines for all three OSes, so this eases getting back a set of results from each OS, which is why only one of the Windows jobs (chosen arbitrarily) uploads.